contributing-device-support.md

```diff
@@ -246,14 +246,27 @@ See [this](https://github.com/balena-io/contracts/pull/296) as an example to bas
 
 For private device types, balena will make the necessary changes when supplied with the hardware contract.
 
-## Step 4: Maintaining the repository and OS updates
+## Step 4: Setting up automated testing
+
+In order to have a balenaOS image published to balena cloud, it must first pass our [automated test suite](./tests).
+This involves the following steps:
+
+1. order and set up an [autokit](https://github.com/balena-io-hardware/autokit-assembly-doc/tree/master?tab=readme-ov-file#autokit-assembly-and-bom) test harness, and connect it to the test board that the new board support is being done for
+2. Follow the [getting started guide](https://balena-os.github.io/leviathan/pages/Getting-Started/quickstart.html#quick-start-guides) to set up our test environment, and then the [guide to running the tests with the autokit](https://balena-os.github.io/leviathan/pages/Getting-Started/quickstart/quickstart-autokit.html)
+3. Once the autokit setup is confirmed to be working locally, it must be inserted into our CI/CD pipeline. This requires contacting the balena team, and transferring the autokit setup into our test fleet
+4. Once the balena team has hooked the test setup into our CI/CD pipline, the board support repository will now have required os-testing checks on PR's to the board support repository
+5. As the board support repository is in the balena-os github org, it will recieve automatic pull requests with updates of meta-balena as new releases of meta-balena get done. Once one of these PR's merge with the tests passing, the device type will be automatically deployed to balena clouds production environment
```

Comment:
```suggestion
5. The board support repository being part of 1balena-os` GitHub org means it would effectively receive automatic updates from meta-balena to keep BalenaOS updated. Each update PR will run the tests on the autokit, which will merge when the tests pass. On merge, the balenaOS release will then be automatically deployed to the balenaCloud production environment.
```